.github/workflows/e2e.yml

```diff
@@ -3,7 +3,11 @@ name: SDK E2E Tests
 on:
   push:
     branches: [main]
+    paths-ignore:
+      - "ui/**"
```

Comment:
Is `ui` the folder for the cloud UI or the dev server?  I'm wondering if the UI folders be:

- `ui-cloud/`
- `ui-dev/`

respectively to clear things up?

Reply:
The `ui` folder will at first only contain the Cloud UI, but soon we'll move the Dev Server UI into that folder. Igor has already prepped Cloud UI to support multiple UIs